parser/include/parser/sourcemanager.h

```diff
@@ -50,9 +46,19 @@ class SourceManager
    * not in the cache yet then a model::File entry is created, persisted in the
    * database and placed in the cache. If the file doesn't exist then it returns
    * nullptr.
+   * @param path_ The file path to look up.
    */
   model::FilePtr getFile(const std::string& path_);
 
+  /**
+   * This function returns a pointer to the corresponding model::File objects
+   * based on the given beta_ filter. The objects are read from a cache.
+   * Uncached files are ignored.
+   * @param beta_ A filter functor iterated over the cached model::File objects.
```

Comment:
What are cached files? The `_files` container is filled up when starting the parser, so it always contains all files. Please check this statement, but I think it works like that.

Reply:
> The _files container is filled up when starting the parser, so it always contains all files.
Exactly, that is why we named it a cache. Since it contains all files, which were in the DB when the `SourceManager` was instantiated or were later added through the `SourceManager`.
It does not contain files which were added to the DB directly or thorough another process. That is an in-memory cache of the File table in my understanding.

I think we mean the same thing here. Please suggest a better naming if you think *caching* is confusing here in the documentation.